src/bun.js/net.exports.js

```diff
@@ -53,9 +53,255 @@ export function isIP(s) {
   return 0;
 }
 
+const { Bun, createFIFO } = import.meta.primordials;
+const { connect: bunConnect } = Bun;
+const { Duplex } = import.meta.require("node:stream");
+
+export class Socket extends Duplex {
+  static #Handlers = {
+    close: Socket.#Close,
+    data(socket, buffer) {
+      const self = socket.data;
+      self.bytesRead += buffer.length;
+      if (self.#readQueue.isEmpty()) {
```

Comment:
since we access this getter twice, lets make it a local variable

Reply:
Done (two sites) - https://github.com/oven-sh/bun/pull/1701/commits/372ee4ef35199d466adcc92602d83f5e851961a4